### ISSUE TITLE
val_acc

### ISSUE THREAD

**Issue by Leo-think** [teddykoker/image-gpt#12]
Hi,Teddy, I first used your model 'models/mnist_gen. ckpt' to test, I wanted to get loss and val_acc, so I set the Classify to True, and finally I got the follwing result.
command：` python3 src/run.py --dataset mnist test models/mnist_gen.ckpt configs/xxs_clf.yml
`

result：`'test_loss': tensor(2.9475, device='cuda:0'),
 'val_acc': tensor(0.0787, device='cuda:0')`

Then I pre-trained a model using the MNIST dataset according to the steps you gave, and made some fine-tuning. Finally, I tested the model and got the follwing result. 
command：` python3 src/run.py --dataset mnist test models/epoch=468.ckpt configs/xxs_clf.yml
`

result：`'test_loss': tensor(2.9911, device='cuda:0'),
 'val_acc': tensor(0.1020, device='cuda:0')`

So I wrote this issue to find out what causes val_acc to be so low.

**Comment by teddykoker** [comment]
Hi,

Using the pretrained *generative* model will not give you a good *classification* accuracy because it was not trained for that task. You should however see a decent classification accuracy once you finetune the model for classification. Could you share exactly the steps you used to do this?

**Comment by teddykoker** [comment]
Running:
```bash
python src/run.py --dataset mnist train configs/xxs_clf.yml --pretrained=models/mnist_gen.ckpt
```
(Halting it with `Ctrl-C` after 2 epochs, you can train for longer if you want I just wanted to show it works)

and then:
```bash
python src/run.py --dataset mnist test logs/xxs_clf_mnist/version_0/checkpoints/epoch=1.ckpt configs/xxs_clf.yml

```

It achieved:
```
{'test_acc': tensor(0.9709, device='cuda:0'),
 'test_loss': tensor(0.5110, device='cuda:0')}
```

You can download my model here: [epoch=1.ckpt.zip](https://github.com/teddykoker/image-gpt/files/7014957/epoch.1.ckpt.zip)
